apps/block_scout_web/lib/block_scout_web/templates/smart_contract/_functions.html.eex

```diff
@@ -36,7 +36,7 @@
                 ) %></h3>
 </div>
 <% end %>
-<%= for {function, counter} <- Enum.with_index(@read_only_functions ++ @read_functions_required_wallet, 1) do %>
+<%= for {function, counter} <- Enum.with_index(if smart_contract_native, do: @read_functions_required_wallet ++ @read_only_functions, else: @read_only_functions ++ @read_functions_required_wallet) do %>
```

Comment:
good catch, I remove it during the if condition refactor. 
With the last commit I restore it but this will not sort out any difference in the methods list. I will keep it so we will not modify in any way the previous logic for the solidity contracts.